adi/jesd_internal.py

```diff
@@ -5,7 +5,94 @@
 from .sshfs import sshfs
 
 
-class jesd:
+class eye_scan(object):
+    _jesd_es_duration_ms = 100
+    _jesd_prbs = 7
+
+    _half_rate = {"mode": "Half Fate", "scale": 0.004}
+    _quarter_rate = {"mode": "Quarter Rate", "scale": 0.001}
```

Comment:
For quarter rate we need to scale until the API updates the next time.
So scale should be 4.0 here

Reply:
Fixed